cmd/skopeo/main.go

```diff
@@ -113,6 +109,8 @@ func createApp() (*cobra.Command, *globalOptions) {
 		tagsCmd(&opts),
 		untrustedSignatureDumpCmd(),
 	)
+	// add the cobra completion command to auto generate shell completion scripts.
+	completion.AddCompletionCommand(rootCommand)
```

Comment:
https://github.com/spf13/cobra 1.3.0 now provides
```go
rootCmd.CompletionOptions.HiddenDefaultCmd = true
```

Do we really need a special case command, different from the upstream one? I see some value in using the library as the upstream intends it (and, sure, some value in being consistent among all c/common users, but the latter seems easier to change.)

Reply:
The upstream command was only added after I already had implemented it in podman, unfortunately the command is not compatible with the podman completion command. The podman command offers the `--file` and `--no-desc` option while the upstream command only has `--no-description`. Also the podman one adds an extra line that the file was generated automatically: https://github.com/containers/common/blob/1bc8353fc42e4a766da619a0b4134a2cfbcd9652/pkg/completion/command.go#L93

I would like to use the upstream command but I also do not want to break podman. I cannot modify the upstream command to accept the current podman options.
We do not have this problem in skopeo of course but I think having a consistent command across skopeo,buildah,podman is valuable.
Alternatively we can start using the upstream command here and change the podman command with v5.0. This would reduce at least some maintenance.

Reply:
I will change it to the upstream one. This is better in the long run and more consistent with other applications which use cobra. I can change podman with the next major release and I doubt that anybody will care if the option is renamed.

The better thing about using the upstream one is that `skopeo completion bash --help` would already show information on how to load the script. I think in this case I could just add this as note in the install guide.